www/templates/main-goals.html

```diff
@@ -54,6 +55,7 @@
               <span ng-show="profile.items.currentPet" class="current-pet Pet-{{profile.items.currentPet}}"></span>
             </div>
           </figure>
+          <!--Habitica jade to html code ends here-->
```

Comment:
Also does this mean that we now need to run `gulp` for `e-mission` too?
If so, that needs to be put into the README